exonum-java-binding/core/rust/src/proxy/runtime.rs

```diff
@@ -163,6 +163,34 @@ impl Runtime for JavaRuntimeProxy {
         })
     }
 
+    fn initiate_resuming_service(
+        &self,
+        context: ExecutionContext,
+        spec: &InstanceSpec,
+        parameters: Vec<u8>,
+    ) -> Result<(), ExecutionError> {
+        let serialized_instance_spec: Vec<u8> = spec.to_bytes();
+
+        jni_call_default(&self.exec, |env| {
```

Comment:
Shan't it be `jni_call_transaction`? Same with `initiate_starting` — @ivan-ochc — does it work as expected?

Reply:
According to documentation of `CallType` we should handle it with `jni_call_transaction`, yes.
```rust
/// Type of a call to a service.
#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
#[serde(tag = "call_type", rename_all = "snake_case")]
pub enum CallType {
    /// Service initialization or resuming.
    Constructor,
    /// Service method.
    Method {
.....
```